content/react/concepts/routing/routing.md

```diff
@@ -21,27 +21,41 @@ React Router is a third-party library that allow components to be used as routed
 - `react-router-native`: React Native version
 - `react-router-dom`: Recommended for web applications
 
-Install with `npm`:
+Install the newest version with `npm`:
 
 ```
-npm install react-router-dom
+npm install react-router-dom@latest
 ```
 
+The newest version of React Router is v6, which comes with many breaking changes including:
```

Comment:
What do we think about linking out to the official overview page for React Router v6? Like so: 

The newest version of React Router is [v6](https://reactrouter.com/docs/en/v6/getting-started/overview), which comes with many breaking changes including:

Reply:
Ditto on this point